Modules/FindNetCDF.cmake

```diff
@@ -178,11 +178,16 @@ if( NOT _val MATCHES "^(yes|no)$" )
 endif()
 if( _val MATCHES "^(yes)$" )
   set(NetCDF_PARALLEL TRUE CACHE STRING "NetCDF has parallel IO capability via pnetcdf or hdf5." FORCE)
-  find_package(MPI REQUIRED COMPONENTS C)
 else()
   set(NetCDF_PARALLEL FALSE CACHE STRING "NetCDF has no parallel IO capability." FORCE)
 endif()
 
+
+if(NetCDF_PARALLEL)
+  # Searches for all enabled languages
+  find_package(MPI)
```

Comment:
Shouldn't there be REQUIRED here. Otherwise, it is just going to fail elsewhere when linking and we will have to scratch our heads (well, not us!)
May be have an error trap, 
```
if(NOT MPI_FOUND)
  message(FATAL_ERROR "NetCDF has parallel enabled, but was unable to find MPI")
endif()
```